### PR TITLE
chore(modern-cmake): release amp-embedded-infra-lib 2.2.1

### PR DESCRIPTION
**Pull request by rjaegers**
:robot: I have created a release *beep* *boop*
---

### Bug Fixes

* Use gh cli to publish release assets ([#184](https://github.com/philips-software/amp-embedded-infra-lib/issues/184)) ([9de4c45](https://github.com/philips-software/amp-embedded-infra-lib/commit/9de4c453be231feea5f973968ad908355d8c6e43))

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  ".": "2.2.0"
```
Comment (EkelmansPh) [inline .release-please-manifest.json:2]: That's a lot of places to update versions. This done automatically or manually?

**Review by EkelmansPh — APPROVED** [review]

File: .release-please-manifest.json (hunk 1 above)
Comment (rjaegers) [reply .release-please-manifest.json:2]: That is done automatically by the release-please action :).

Comment (EkelmansPh) [reply .release-please-manifest.json:2]: What I hoped for, nice :)

**Comment by rjaegers** [comment]
:robot: Release is at https://github.com/philips-software/amp-embedded-infra-lib/releases/tag/amp-embedded-infra-lib-v2.2.1 :sunflower:

**Pull request merged**